src/tbp/interactive/data.py

```diff
@@ -100,6 +103,84 @@ def create_mesh(self, obj_name: str) -> Mesh:
         return obj
 
 
+class _MontyShimUnpickler(pickle.Unpickler):
+    """Unpickler that shims out ``tbp.monty`` classes with lightweight dummies.
+
+    Any class reference from a module path starting with ``tbp.monty`` is
+    redirected to a dynamically created dummy class. This allows deserialization
+    of checkpoints that reference ``tbp.monty`` types even when the actual
+    package is not installed, while still making object attributes
+    (e.g., ``.pos``, ``.x``) accessible.
+
+    Dummy classes are cached by ``(module, name)`` so that repeated lookups
+    return the same type.
+    """
+
+    _cache = {}  # cache dummy classes
+
+    def find_class(self, module, name):
+        if module.startswith("tbp.monty"):
+            key = (module, name)
+            cls = self._cache.get(key)
+            if cls is None:
+                cls = type(name, (), {})
+                self._cache[key] = cls
+            return cls
+        return super().find_class(module, name)
+
+
+class PretrainedModelsLoader:
+    """Load Monty pretrained Models as point cloud Vedo object."""
+
+    def __init__(self, data_path: str, lm_id: int = 0, input_channel: str = "patch"):
+        """Initialize the loader.
+
+        Args:
+            data_path: Path to the `model.pt` file holding the pretrained models.
+            lm_id: Which learning module to use when extracting the pretrained graphs.
+            input_channel: Which channel to use for extracting the pretrained graphs.
+        """
+        self.path = data_path
+        models = self._torch_load_with_optional_shim()["lm_dict"][lm_id]["graph_memory"]
+        self.graphs = {k: v[input_channel]._graph for k, v in models.items()}
+
+    def _torch_load_with_optional_shim(self, map_location="cpu"):
+        """Load a torch checkpoint with optional fallback for tbp.monty shimming.
+
+        Try a standard torch.load first (weights_only=False because we need objects).
+        If tbp.monty isn't installed and the checkpoint references it, optionally
```

Comment:
Is there a reason tbp.monty wouldn't be installed if using these plotting tools? Was this something you came across?